tests/typegen/strats/test_inline.py

```diff
@@ -336,18 +336,6 @@ def test_callables():
         "int",
     ]
 
-    traced.loc[len(traced.index)] = [
-        str(resource_path),
-        c_clazz_module,
-        c_clazz,
-        "inner",
-        0,
-        TraceDataCategory.CLASS_MEMBER,
-        "a",
-        None,
-        "int",
-    ]
```

Comment:
I think it should be kept. While inner() does not contain the class instance as parameter, it changes a class member of a class instance (from outer).

Reply:
I see, the data of class member of a already exists.